site/components/buttons/DeleteDisruptionButton.tsx

```diff
@@ -25,7 +40,7 @@ const DeleteDisruptionButton = ({
             {showDeleteModal && (
                 <DeleteConfirmationPopup
                     entityName={isTemplate ? "the template" : "the disruption"}
-                    deleteUrl={`/api/delete-disruption${isTemplate ? "?template=true" : ""}`}
+                    deleteUrl={`/api/delete-disruption${getQueryParams(isTemplate == "true", returnPath)}`}
```

Comment:
```suggestion
                    deleteUrl={`/api/delete-disruption${getQueryParams(isTemplate == "true", returnPath)}`}
```deleteUrl={`/api/delete-disruption${getQueryParams(isTemplate === "true", returnPath)}`}

Reply:
done